molecule/broker-scale-up/converge.yml

```diff
@@ -25,11 +25,31 @@
         fail_msg: "Number of brokers are not 3"
 
     - name: Run zookeeper shell to get broker ID's
-      # Kraft:TODO: get cluster ID from different method for kraft based scenarios
       shell: >
           /usr/bin/zookeeper-shell zookeeper1:2182  -zk-tls-config-file  /etc/kafka/zookeeper-tls-client.properties ls /brokers/ids
       register: zk_output
       when: not ('kafka_controller' in groups.keys() and groups['kafka_controller'] | length > 0)
+
+    - name: Get broker ID's
+      shell: >
+          kafka-metadata-quorum --command-config /etc/kafka/client.properties --bootstrap-server kafka-broker1:9091 describe --status | grep CurrentObservers | awk '{print $NF}'
+      register: zk_output
```

Comment:
Shouldn't we name it like controller_output?

Reply:
i'll fix this in next PR.